imagersite/templates/imagersite/base.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
You don't have to store this template in `imagersite`. In fact, I'd even go so far as to say you shouldn't because this is going to be the base template for the rest of your site so you shouldn't need to extend from `imagersite/base.html`. It'd be easier for you to just extend from `base.html`